harvardcards/apps/flash/api/card.py

```diff
@@ -44,9 +44,14 @@ def edit(request):
         else:
             base_url = deck.get_absolute_url()
 
+        if add_another:
+            base_url = deck.get_add_card_url()
+            goto_url = "{0}?deck_id={1}".format(base_url, deck.id)
+        else:
+            card_url = "{0}?card_id={1}".format(base_url, card.id)
```

Comment:
Shouldn't there be a value for ```goto_url``` in the *else*? If the user chooses to just submit the card instead of submit and add another, this will generate an error since the goto_url variabledoesn't have a value.